build-tools/packages/build-tools/package.json

```diff
@@ -41,6 +41,7 @@
 	"dependencies": {
 		"@fluid-tools/version-tools": "workspace:~",
 		"@fluidframework/bundle-size-tools": "workspace:~",
+		"@fluidframework/test-runtime-utils": "^1.3.7",
```

Comment:
This creates a circular dependency between build-tools and client. That's not really allowed.

Reply:
@WayneFerrao we likely don't really need this. Presumably you're using it to set up your Mocha config, but we can just set one up manually. You can probably use the config in the eslint package as reference for this.